apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -148,21 +181,37 @@ private string ValidatedDataSource(string dataSource)
 
         /// <summary>
         /// The TCP Host name to connect to Spanner. If not supplied in the connection string, the default
-        /// host will be used.
+        /// host will be used. If the SPANNER_EMULATOR_HOST environment variable is set, the host specified
+        /// in its value will be used as the default host.
         /// </summary>
         public string Host
         {
-            get => GetValueOrDefault(nameof(Host), SpannerClient.DefaultEndpoint.Host);
+            get
+            {
+                if (ContainsKey(nameof(Host)))
```

Comment:
So, if the Host is explicitly set it takes presedence over the emulator?

Reply:
No longer an issue since the implementation has changed.